src/main/webapp/js/ListPoems.js

```diff
@@ -0,0 +1,48 @@
+import { CreateElement } from './Common.js';
+/**
+ * List poems
+ */
+class ListPoems {
+  constructor() {
+    this.data = null;
+  }
+  /**
+   * Fetch a list of poems from the server
+   * @returns A JSON list of poems
+   */
+  async GetPoemData() {
+    const response = await fetch('/poem');
```

Comment:
lowercase the first characted in the function name. So getPoemData instead of GetPoemData